# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: /space/hudson/hudson/main/core/src/main/java
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: hudson/Proc.java
--- hudson/Proc.java Base (BASE)
+++ hudson/Proc.java Locally Modified (Based On LOCAL)
@@ -14,6 +14,7 @@
 import java.util.concurrent.Future;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import org.jvnet.winp.WinProcess;
 
 /**
  * External process wrapper.
@@ -163,6 +164,14 @@
          * Destroys the child process without join.
          */
         private void destroy() {
+            //on windows
+            if(File.pathSeparatorChar==';'){
+                try {
+                    WinProcess p = new WinProcess(proc);
+                } catch (Exception ex) {
+                    //will be thrown
+                }
+              }
             ProcessTreeKiller.get().kill(proc,cookie);
         }
 
